images/runtime/dotnetcore/2.1/Dockerfile.base

```diff
@@ -0,0 +1,46 @@
+# dotnet tools are currently available as part of SDK so we need to create them in an sdk image
+# and copy them to our final runtime image
+FROM mcr.microsoft.com/dotnet/core/sdk:2.1.802 AS tools-install
+RUN dotnet tool install --tool-path /dotnetcore-tools dotnet-sos
+
+FROM oryx-run-base AS dotnetcore-original
```

Comment:
We could have based on .NET Core team's `runtime-deps` image (so we can avoid the content from lines 8 to 25), but we wanted to keep the base image consistent across our stack. .NET Core team's base images are based on either `stretch-slim` or `buster-slim`